AMPnet/ampnet-crowdfunding-frontend
# [Fixes AB#566] Edit project screen

To test:
Test if ROI works properly
Test if all data is being updated properly
Check if everything is linked correctly in form
Check if redirecting to from ‘’ path is setup right
Check if manage payments screen redirected from revenue share screen looks good

Done
Added redirecting from ‘’ to ‘offers’ (when we accessed site directly, we would still have that annoying route /dash where we would show offers but active link wasn’t highlighted)
Added placeholder image for project in project details if project doesn't have img defined
Changed ROI to be 0 by default, if user doesn’t change value from 0 it is hidden on offers list and offer details screen
